packages/react-router/src/root.ts

```diff
@@ -0,0 +1,2 @@
+export const rootRouteId = '__root__' as const
```

Comment:
Looks like its just the import for this in `src/fileRoute` that needs to be removed.

Reply:
Fixed these

Reply:
Thx